submission-stages1.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Submission stages
```

Comment:
You could just replace the existig submission-stages page - why create a new one?